zabbix-monitoring-cluster/nested/monitoringAgentByIps.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "ipConfigurations": {
+      "type": "array",
+      "metadata": {
+        "description": "ipConfigurations info."
+      }
+    },
+    "serverIp": {
+      "type": "string",
+      "metadata": {
+        "description": "server ip."
+      }
+    },
+    "_artifactsLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+      },
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/zabbix-monitoring-cluster"
+    }
+  },
+  "variables": {
+    "vmidPrefix": "[concat('/subscriptions/',subscription().subscriptionId,'/resourceGroups/',resourceGroup().name,'/providers/Microsoft.Compute/virtualMachines/')]",
```

Comment:
Just checked that resourceId requires a 'resourceName1' name to be set. But here 'vmidPrefix' is terminated with 'resourceType' only, it isn't meant to be a full id.

It is used for 'extracting' VM name later on Line#48.